README.md

```diff
@@ -0,0 +1,52 @@
+# Clangen Docs
+This repository contains the documentation for Clangen.
+
+## Contributing
+
+Thank you for your interest in contributing to Clangen's documentation! To make a change in the documentation, you just have to change the corresponding `.md` file in the `/docs` folder. You can also create a new file in the folder and it will automatically create a new page in the documentation.
+
+## Build Instructions
+
+> [!IMPORTANT]
+> If you just want to contribute documentation, you don't **have** to do this. Building the documentation just creates a local copy of the documentation website on your computer, which can be useful for previewing your changes, but isn't necessarily required.
```

Comment:
Is building better than utilizing the `serve` command?  I wasn't sure of the difference when reading the documentation.

Reply:
I totally forgot that there was a serve command (things that happen when you just go off your memory instead of looking at the docs) 🤦 I will update the readme.

Reply:
Done!